src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import './vars';
+@import './extends';
+@import './mixins';
+@import './keyframes';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  @extend %stopwatch-flex-property;
+}
+
+.stopwatch {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  color: #000;
+
+  @extend %stopwatch-flex-property;
+  @include stopwatch-border-style(1vmin, dotted, 50%);
```

Comment:
The `stopwatch-border-style` mixin is being used here, but it's not clear what it does from the name alone. Consider adding a comment above where the mixin is defined to explain what it does.